nengo_loihi/builder.py

```diff
@@ -326,12 +333,12 @@ def build_ensemble(model, ens):
     gain, bias, max_rates, intercepts = get_gain_bias(
         ens, rng, model.intercept_limit)
 
-    if isinstance(ens.neuron_type, nengo.Direct):
-        raise NotImplementedError()
-    else:
+    if isinstance(ens.neuron_type, (nengo.LIF, nengo.SpikingRectifiedLinear)):
```

Comment:
This prohibits someone from adding their own build hook for a neuron type. Can we leave this as before? The builder should raise an error if it doesn't find a build hook for the neuron type in question.

Reply:
Yeah I think we can leave this as before as the issue was more with the splitter allowing you to get to this point with neuron types we didn't expect. By the same token, since we don't implement a builder to `nengo.Direct` do we even need this block?

Reply:
Yeah, I was wondering about that last part too. Probably not. I just copied the builder from Nengo and modified it, so this is just a remnant of that.

Reply:
Added a fixup for this.